README.md

```diff
@@ -87,11 +87,12 @@ criterion             | description                                           |
 `TimeLimit(t=0.5)`    | Stop after `t` hours                                  |
 `NumberLimit(n=100)`  | Stop after `n` loss updates (excl. "training losses") |
 `NumberSinceBest(n=6)`| Stop after `n` loss updates (excl. "training losses") |
-`Threshold(value=0.0)`| Stop when `loss < value`                              | 
+`Threshold(value=0.0)`| Stop when `loss < value`                              |
 `GL(alpha=2.0)`       | Stop after "Generalization Loss" exceeds `alpha`      | ``GL_α``
 `PQ(alpha=0.75, k=5)` | Stop after "Progress-modified GL" exceeds `alpha`     | ``PQ_α``
 `Patience(n=5)`       | Stop after `n` consecutive loss increases             | ``UP_s``
 `Disjunction(c...)`   | Stop when any of the criteria `c` apply               |
+`Warmup(c, n)`        | Wait for `n` loss updates before checking criteria `c`|
```

Comment:
Thanks for the doc update!

I think for consistency here and with the control wrappers in [IterationControl.jl](https://github.com/JuliaAI/IterationControl.jl#controls-provided) (which extends this package) the second argument `n` should be a keyword argument. For example, we have there `IterationControl.skip(control; predicate=1)` to wrap a control `control`.